grr/server/grr_response_server/gui/ui/store/client_page_facade.ts

```diff
@@ -428,6 +429,17 @@ export class ClientPageStore extends ComponentStore<ClientPageState> {
                   this.httpApiService.addClientLabel(clientId, label)),
           tap(() => this.fetchClient()),
           ));
+
+  // An effect to remove a label from the selected client
+  removeClientLabel(label: string): Observable<HttpResponse<{}>> {
```

Comment:
Functions that trigger change should only return void in NgRx (our store implementation). Instead of returning an observable here, create an observable as member of the Store. Then, when deletion of the label is completed, update the state which should then trigger emission of the dedicated observable. E.g.: the observable that tracks flow creation state.